spiffworkflow-backend/src/spiffworkflow_backend/routes/health_controller.py

```
@@ -1,9 +1,31 @@
-from flask import make_response
+from flask import current_app, request
```

Flake8 (7.2.0)

[error] 'flask.request' imported but unused

(F401)